extensions/sql-database-projects/src/test/dialogs/addDatabaseReferenceDialog.test.ts

```diff
@@ -29,7 +29,7 @@ describe('Add Database Reference Dialog', () => {
 		const dialog = new AddDatabaseReferenceDialog(project);
 		await dialog.openDialog();
 
-		should(dialog.dialog.okButton.enabled).equal(false);
+		should(dialog.dialog.okButton.enabled).equal(true, 'Ok button should be enabled since initial type of systemDb has default values filled');
```

Comment:
@kisantia @Benjin This look good to you? When I open the dialog the "Add References" button is initially enabled so this seems correct. It was due to the race condition stuff I'm partially fixing here that seems to have caused it to be set as disabled initially before.

Reply:
yup looks good. Can you also remove the trying to enable the ok button and checking that it got enabled on lines 34-35 since they aren't needed anymore? Thanks!